scripts/o3de/ExportScripts/export_source_built_project.py

```diff
@@ -230,10 +255,10 @@ def parse_args(o3de_context: exp.O3DEScriptExportContext):
         parser.add_argument('-abp', '--asset-bundling-path', type=pathlib.Path, default=None,
                             help="Designates where the artifacts from the asset bundling process will be written to before creation of the package. If not specified, default is <o3de_project_path>/build/asset_bundling.")
         parser.add_argument('-maxsize', '--max-bundle-size', type=int, default=2048, help='Specify the maximum size of a given asset bundle.')
-        parser.add_argument('-nogame', '--no-game-launcher', action='store_true', help='This flag skips building the Game Launcher on a platform if not needed.')
-        parser.add_argument('-noserver', '--no-server-launcher', action='store_true', help='This flag skips building the Server Launcher on a platform if not needed.')
-        parser.add_argument('-noheadless', '--no-headless-server-launcher', action='store_true', help='This flag skips building the Headless Server Launcher on a platform if not needed.')
-        parser.add_argument('-nounified', '--no-unified-launcher', action='store_true', help='This flag skips building the Unified Launcher on a platform if not needed.')
+        parser.add_argument('-nogame', '--no-game-launcher', default=False, action='store_true', help='This flag skips building the Game Launcher on a platform if not needed.')
```

Comment:
If `action='store_true'`, then why do we need a `default=False`? Didn't python complain about this?

Reply:
python didnt complain about it surprisingly enough, but I've run into confusion with args like it already during prior work with it, so I felt it'd be clearer to be explicit by stating both at once.

Reply:
`default=False` is unnecessary and redundant. You either use `action='store_true'` or `action='store_false'` dependent on what you want the default result to be. (If its `action='store_false'` then the default value is True. ) And if you set `default=True`, then this is a useless argument since you cannot never change the value. I think it should be removed

Reply:
makes sense. will remove